lib/convert.js

```diff
@@ -87,11 +95,17 @@ var convertParametersInner = function(change, convertString, param) {
           convertString = convertString.replace(reg, tempParamKey);
         } else {
           if (change == '#') {
-            tempParamKey = tempParamKey.toString().replace(/"/g, '\\\"');
-            tempParamKey = tempParamKey.replace(/'/g, '\\\'');
-            
-            convertString = convertString.replace(reg, "'" + tempParamKey + "'");
-          } else if (change == '$') {
+            if (change == '#') {

```

Comment:
I think there is no need to use "if (change == '#') {" twice. this condition is used line 97 already.

Reply:
Of course you are right! This is all because of the rush. Thank. Corrected.

Reply:
And I slightly corrected the indentation.